proto/regen/ecocredit/v1alpha2/events.proto

```diff
@@ -6,8 +6,6 @@ import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
 import "google/protobuf/timestamp.proto";
 
-option go_package = "github.com/regen-network/regen-ledger/x/ecocredit";
-
```

Comment:
We should delete the entire `v1alpha2` folder. This version was never used in production.